assets/airports/eddl.json

```diff
@@ -0,0 +1,3131 @@
+            "icao": "DODEN3H",
+            "name": "Doden Three Hotel",
+            "rwy": {
+                "EDDL23R": []
+            },
+            "body": ["DL240", "DL241", ["DL242", "A150+"], "ELBAL", "BETZO"],
+            "exitPoints": {
+                "DODEN": [["DODEN", "A250+"]]
+            },
+            "draw": [
+                ["DODEN"]
+            ]
+        },
+        "DODEN6J": {
+            "icao": "DODEN6J",
+            "name": "Doden Six Juliett",
+            "rwy": {
+                "EDDL05L": []
+            },
+            "body": ["DL050", "DL058", "BAM", ["ELBAL", "A150+"], "BETZO"],
+            "exitPoints": {
+                "DODEN": [["DODEN", "A250+"]]
+            },
+            "draw": [
+                ["BAM", "ELBAL", "BETZO", "DODEN"]
+            ]
+        },
+        "DODEN6Z": {
+            "icao": "DODEN6Z",
+            "name": "Doden Six Zulu",
+            "rwy": {
+                "EDDL05R": []
+            },
+            "body": ["DL050", "DL058", "BAM", ["ELBAL", "A150+"], "BETZO"],
+            "exitPoints": {
+                "DODEN": [["DODEN", "A250+"]]
+            },
+            "draw": [
+                ["DODEN"]
+            ]
+        },
+        "DODEN9T": {
+            "icao": "DODEN9T",
+            "name": "Doden Nine Tango",
+            "rwy": {
+                "EDDL23L": []
+            },
+            "body": ["DL239", "DL241", ["DL242", "A150+"], "ELBAL", "BETZO"],
+            "exitPoints": {
+                "DODEN": [["DODEN", "A250+"]]
+            },
+            "draw": [
+                ["DL244", "DL242"],
+                ["DODEN"]
+            ]
+        },
+        "GMH3H": {
+            "icao": "GMH3H",
+            "name": "Germinghausen Three Hotel",
+            "rwy": {
+                "EDDL23R": []
+            },
+            "body": ["DL240", "DL241"],
+            "exitPoints": {
+                "GMH": ["GMH"]
+            },
+            "draw": [
+                ["DL240", "DL241"],
+                ["ELBAL", "GMH"]
+            ]
+        },
+        "GMH8Y": {
+            "icao": "GMH8Y",
+            "name": "Germinghausen Eight Yankee",
+            "rwy": {
+                "EDDL05L": []
+            },
+            "body": ["DL050", "DL058", "BAM", "ANAVI"],
+            "exitPoints": {
+                "GMH": ["GMH"]
+            },
+            "draw": [
+                ["DL050", "DL058", "BAM", "ANAVI", "GMH"]
+            ]
+        },
+        "GMH9T": {
+            "icao": "GMH9T",
+            "name": "Germinghausen Nine Tango",
+            "rwy": {
+                "EDDL23L": []
+            },
+            "body": ["DL239", "DL241"],
+            "exitPoints": {
+                "GMH": ["GMH"]
+            },
+            "draw": [
+                ["DL239", "DL240"],
+                ["GMH"]
+            ]
+        },
+        "GMH9Z": {
+            "icao": "GMH9Z",
+            "name": "Germinghausen Nine Zulu",
```

Comment:
Line Niner Niner Niner :wink: :
We should probably use the official "Niner"

Reply:
999 - what's your emergency?